packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -44,92 +44,128 @@ abstract contract CallBatch is Script {
         calls.push(GovernanceMessage.Call(to, data));
     }
 
+    function write(string memory indent, string memory line) private {
+        vm.writeLine(outputFile, string(abi.encodePacked(indent, line)));
+    }
+
+    function writeKV(
+        string memory indent,
+        string memory key,
+        string memory value,
+        bool terminal
+    ) private {
+        string memory comma = terminal ? "" : ",";
+
+        bytes memory line = abi.encodePacked(
+            '"',
+            key,
+            '": "',
+            value,
+            '"',
+            comma
+        );
+        write(indent, string(line));
+    }
+
+    function writeObjectBody(string memory indent, string[2][] memory kvs)
+        private
+    {
+        for (uint256 i = 0; i < kvs.length; i++) {
+            writeKV(indent, kvs[i][0], kvs[i][1], i == kvs.length - 1);
+        }
+    }
+
+    function writeArrayOpen(string memory indent, string memory key) private {
```

Comment:
why is this in CallBatch? needs to be rebased out, this should be in JSONWriter, no?

Reply:
Yeah, ignore this file. It will be rebased.